p2p/raintree/network.go

```diff
@@ -78,8 +103,21 @@ func (n *rainTreeNetwork) networkBroadcastAtLevel(data []byte, level uint32, non
 	return nil
 }
 
+// Cleanup layer is just another send left / right
+// This layer happens at level -1
+func (n *rainTreeNetwork) cleanupLayer() []target {
+	// TODO (Team) unhappy path where the left / right nodes are down
+	// 			   (continue to search left and right until you have a hit)
+	return n.getCleanupTargets()
+}
+
+// Redundancy layer is simply a redundant send to the original +1/3 && -1/3
+func (n *rainTreeNetwork) redundancyLayer() (level int32, msgLevel int32) {
+	return int32(n.peersManager.maxNumLevels), 0
```

Comment:
<img width="883" alt="Screenshot 2022-12-01 at 7 38 38 PM" src="https://user-images.githubusercontent.com/1892194/205209570-54590e81-4e97-44e7-a7ab-7cfb045bcdbc.png">

You are correct. I believe this was simply overlooked and would simply consider this incomplete.

The changes we need to make is:
1. Potentially make this is a UDP message (not TCP), but seems like that's a lot of added work.
2. Make sure this results in a send to the receiving node (not a broadcast)
3. Make the change to have this be O(N)